GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -20,6 +21,8 @@ internal class RevisionGridMenuCommands : MenuCommandsBase
 
         private readonly RevisionGridControl _revisionGrid;
 
+        public event EventHandler OnSwitchBetweenArtificialCommits;
```

Comment:
```suggestion
		/// <summary>
		///  Occurs whenever a user toggles between HEAD and artificial commits
		///  via a shortcut command.
		/// </summary>		
        public event EventHandler ArtificialOrHeadCommitSelected;
```

Reply:
The name `ArtificialOrHeadCommitSelected` is too unprecise IMO. Renamed to `ToggledBetweenArtificialAndHeadCommits`.